src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/unallocatedcases/GetUnallocatedCasesByTeamTests.kt

```diff
@@ -154,6 +163,11 @@ class GetUnallocatedCasesByTeamTests : IntegrationTestBase() {
         Triple("ZZZZZZZ", true, true),
       ),
```

Comment:
workforceAllocationsToDelius.userHasAccessToAllCases Method may need updating in test   `Get unallocated cases by team where all cases are LAO cases`() {

Reply:
Done. Call wasn't needed in that instance